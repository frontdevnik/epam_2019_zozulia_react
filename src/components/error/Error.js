import React from 'react';

export default function Error(props) {
  console.log(props.error);
  return <h1 style={ { textAlign: 'center' } }>Sorry, the service is not available now, try visiting later</h1>;
}
