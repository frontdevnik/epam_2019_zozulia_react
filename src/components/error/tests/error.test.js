import React from 'react';
import { shallow } from 'enzyme';
import Error from '../Error';

describe('<Error />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Error />);
  });

  it('should render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should render text correctly', () => {
    expect(wrapper.find('h1').text()).toEqual('Sorry, the service is not available now, try visiting later');
  });
});
