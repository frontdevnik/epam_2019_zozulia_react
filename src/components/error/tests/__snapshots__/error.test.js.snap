// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should render correctly 1`] = `"<h1 style=\\"text-align:center\\">Sorry, the service is not available now, try visiting later</h1>"`;
